bolt/lib/Rewrite/RewriteInstance.cpp

```diff
@@ -1419,6 +1503,10 @@ Error RewriteInstance::discoverRtFiniAddress() {
   }
 
   if (!BC->FiniArrayAddress || !BC->FiniArraySize) {
+    // It is still possible to generate profile without fini hook if
+    // InstrumentationSleepTime is set
+    if (opts::InstrumentationSleepTime > 0)
```

Comment:
Since changes involving `opts::InstrumentationSleepTime` are unrelated to the `DT_INIT_ARRAY`, let's create a separate PR for them.